server/src/main/java/org/elasticsearch/search/SearchHit.java

```diff
@@ -167,6 +171,25 @@ public long getVersion() {
         return this.version;
     }
 
+
+    public void setSeqNo(long seqNo) {
+        this.seqNo = seqNo;
+    }
+
+    public void setPrimaryTerm(long primaryTerm) {
+        this.primaryTerm = primaryTerm;
+    }
+
+    /** returns the sequence number of the last modification to the document */
+    public long getSeqNo() {
+        return this.seqNo;
+    }
+
+    /** returns the primary term of the last modification to the document */
+    public long getPrimaryTerm() {
+        return this.primaryTerm;
+    }
```

Comment:
maybe document default return values, if seq_no_primary_term is not set?

Reply:
+1